iphone/Maps/Bookmarks/Categories/Sharing/BookmarksSharingViewController.swift

```diff
@@ -0,0 +1,76 @@
+import SafariServices
+
+final class BookmarksSharingViewController: MWMTableViewController {
+  typealias ViewModel = MWMAuthorizationViewModel
+  
+  @IBOutlet weak var uploadAndPublishCell: UploadActionCell!
+  @IBOutlet weak var getDirectLinkCell: UploadActionCell!
+  
+  @IBOutlet weak var uploadAndPublishDescLabel: UILabel! {
```

Comment:
Set localized text in storyboard via User Defined Runtime Attributes, "localizedText" key